[2024-06-18T17:15:59.759+0545] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-18T17:15:59.792+0545] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: database_backup.backup_task scheduled__2024-06-18T11:20:00+00:00 [queued]>
[2024-06-18T17:15:59.802+0545] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: database_backup.backup_task scheduled__2024-06-18T11:20:00+00:00 [queued]>
[2024-06-18T17:15:59.802+0545] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-18T17:15:59.825+0545] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): backup_task> on 2024-06-18 11:20:00+00:00
[2024-06-18T17:15:59.827+0545] {standard_task_runner.py:63} INFO - Started process 4124 to run task
[2024-06-18T17:15:59.835+0545] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'database_backup', 'backup_task', 'scheduled__2024-06-18T11:20:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/backup.py', '--cfg-path', '/tmp/tmp74c44yha']
[2024-06-18T17:15:59.837+0545] {standard_task_runner.py:91} INFO - Job 17: Subtask backup_task
[2024-06-18T17:15:59.923+0545] {task_command.py:426} INFO - Running <TaskInstance: database_backup.backup_task scheduled__2024-06-18T11:20:00+00:00 [running]> on host PRABAL-10.
[2024-06-18T17:16:00.026+0545] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='prabal' AIRFLOW_CTX_DAG_ID='database_backup' AIRFLOW_CTX_TASK_ID='backup_task' AIRFLOW_CTX_EXECUTION_DATE='2024-06-18T11:20:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-18T11:20:00+00:00'
[2024-06-18T17:16:00.028+0545] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-18T17:16:00.029+0545] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-18T17:16:00.031+0545] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/prabal/bashscripts/backup.sh ']
[2024-06-18T17:16:00.052+0545] {subprocess.py:86} INFO - Output:
[2024-06-18T17:16:00.416+0545] {subprocess.py:93} INFO - mysql: [Warning] Using a password on the command line interface can be insecure.
[2024-06-18T17:16:01.147+0545] {subprocess.py:93} INFO - mysqldump: [Warning] Using a password on the command line interface can be insecure.
[2024-06-18T17:16:01.230+0545] {subprocess.py:93} INFO - mysqldump: Error: 'Access denied; you need (at least one of) the PROCESS privilege(s) for this operation' when trying to dump tablespaces
[2024-06-18T17:16:01.349+0545] {subprocess.py:93} INFO - Backup completed in 0 seconds
[2024-06-18T17:16:01.350+0545] {subprocess.py:93} INFO - Backup Success in location: /home/prabal/backup|Required Space: 0GB, Available Space: 1256.32GB
[2024-06-18T17:16:01.350+0545] {subprocess.py:97} INFO - Command exited with return code 0
[2024-06-18T17:16:01.351+0545] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-18T17:16:01.382+0545] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=database_backup, task_id=backup_task, run_id=scheduled__2024-06-18T11:20:00+00:00, execution_date=20240618T112000, start_date=20240618T113059, end_date=20240618T113101
[2024-06-18T17:16:01.417+0545] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-18T17:16:01.453+0545] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-18T17:16:01.460+0545] {local_task_job_runner.py:222} INFO - ::endgroup::
